packages/frontend/src/components/EditorSettings/FlowTransfer.tsx

```diff
@@ -24,25 +36,20 @@ export function CustomSpinner() {
 }
 
 export default function FlowTransfer() {
+  const inputDescriptionText =
+    'Please enter a valid account on Plumber e.g. me@example.gov.sg'
```

Comment:
nit: this copy might be very confusing - e.g. after the user typed in a valid email, this message still shows up. They might think they have made a mistake.

Maybe we can change the error message to this and not show the input description text at all.